library/std/src/primitive_docs.rs

```diff
@@ -949,6 +957,31 @@ mod prim_str {}
 #[stable(feature = "rust1", since = "1.0.0")]
 mod prim_tuple {}
 
+// Required to make auto trait impls render.
+// See /src/librustdoc/passes/collect_trait_impls.rs:collect_trait_impls
```

Comment:
Elsewhere in the codebase, it looks like paths are expressed without this leading `/`.

```suggestion
// See src/librustdoc/passes/collect_trait_impls.rs:collect_trait_impls
```